endpoints/openrtb2/auction_test.go

```diff
@@ -6146,6 +6146,395 @@ func fakeNormalizeBidderName(name string) (openrtb_ext.BidderName, bool) {
 	return openrtb_ext.BidderName(strings.ToLower(name)), true
 }
 
+func TestValidateOrFillCDep(t *testing.T) {
```

Comment:
Nitpick: Could you update this name to be `TestValidateOrFillCookieDeprecation`

Reply:
done